src/styles/images/index.md.njk

```diff
@@ -25,10 +25,30 @@ If your image represents something physical, such as a letter, document or credi
 
 {{ example({group: "styles", item: "images", example: "representative", html: false, open: false, size: "l"}) }}
 
+
 ## Alt text
 
-All images must have an alt text. You should include a description of the image inside the tag to help users who are unable to see it.
+All images must have alt text. This is a description of an image that tells people what information the image provides. It is read out by screen readers or displayed if an image does not load or if images have been switched off.
+
+Alt text must:
+
+- describe the content and function of the image
+- be specific, meaningful and concise
+- use normal punctuation and end with a full stop
+
+Do not:
+
+- include the name of the photographer or person who created the image
+- start with ‘Image of’, ‘Graphic of’ or ‘Photo of’
+- repeat information from the page
+- include extra information not on the page
 
 {{ example({group: "styles", item: "images", example: "alt-text", html: true, open: true, size: "l"}) }}
 
-If your image is purely decorative and a description would be confusing you should still include the alt text, but leave it empty so it will be ignored by screen readers.
+Some images do not need descriptions. Use `alt=""` if the image is:
+
+- decorative and does not include important content
+- an icon that has a text label
+- part of a link
```

Comment:
I also misunderstood this one at first reading. I assume what is meant here is if a link consists of text and also an image.
It might be good to add to the Do's above that when an image is inside a link it has to have a description, otherwise the image will be empty to screen readers.